docs/schemata/game.yml

```diff
@@ -6,20 +6,103 @@ title: Game
 description: |-
   Loosely based off the example in the README
 
-type: object
-properties:
+definitions:
+  identity:
+    anyOf:
+      - $ref: /schemata/game#/definitions/id
+  id:
+    type: number
+    description: |-
+      this could be a uuid to make it db agnostic
   black_id:
-    $ref: /schemata/player#/properties/id
+    description: black player's id
+    $ref: /schemata/player#/definitions/id
   white_id:
-    $ref: /schemata/player#/properties/id
+    description: white player's id
+    $ref: /schemata/player#/definitions/id
   game_server:
     type: string
+    description: game server id
     example: KGS
   rated:
     type: boolean
+    description: whether this was a rated game (on the server?)
   date_played:
     type: string
+    description: time the game was played
     format: date-time
   game_record:
     type: string
+    description: game record in SGF
     example: <raw sgf string>
+
+links:
+- href: /api/v1/games
+  method: POST
+  description: Report a game result
+  rel: create
+  title: Creation
+  schema:
```

Comment:
OTOH, it does handle a lot of common cases around resources, relationships, etc.